main.py

```diff
@@ -38,6 +52,127 @@
     db_format = json.load(file_handle)
 
 
+def file_size(file_path):
+    """
+    this function will return the file size
+    """
+    if os.path.isfile(file_path):
+        return os.stat(file_path)
+
+
+def writeToFireBase():
+    """
+    This function will write to Firebase with the given buffer.
+    """
+    try:
+        collections = COL_TELEMETRY.document(timestampStr).collections()
+        for col, sensor in zip(collections, SENSORS):
+            for sec in buffer.keys():
+                data_per_timeframe = int(buffer[sec][sensor])
+                col.document("0").update({
+                    str(sec) : data_per_timeframe
+                })
+        buffer.clear()
+        print("Buffer clear")
+    except Exception as e:
+        exc_type, exc_obj, exc_tb = sys.exc_info()
+        fname = os.path.split(exc_tb.tb_frame.f_code.co_filename)[1]
+        print(exc_type, fname, exc_tb.tb_lineno)
+        print(e)
+        
+
+countdownToBufferClear = Timer(60.0, writeToFireBase)
```

Comment:
This timer probably shouldn't be started until first receiving data from the car as someone could be loading up the app just to look at historical data

Reply:
Another leftover from TelemetryServer